### DISCUSSION TITLE
ZMODELPROTOEXTENDER

### DISCUSSION THREAD

**Discussion by Kor-Angar** [dosinabox/g2nr_unofficial_update#458] [General]
топик https://worldofplayers.ru/threads/42117/

**Comment by Kor-Angar** [comment]
dll лежит в папке авторан. я его не нашёл ни в топики zModelProtoExtender ни в менеджере ресурсов. возможно это старая версия файла.

[ZMODELPROTOEXTENDER.zip](https://github.com/dosinabox/g2nr_unofficial_update/files/9078845/ZMODELPROTOEXTENDER.zip)

**Reply by dosinabox** [reply]
актуальный zModelProtoExtender.dll находится в Хуманс-ремастере

**Reply by Kor-Angar** [reply]
я этот файл оттуда и дёрнул :-)

**Comment by Kor-Angar** [comment]
> Куда копируется этот mds?

mds файл компилируется в папку с игрой с сохранением префиксов
![image](https://user-images.githubusercontent.com/49514764/178140282-7f06f89c-ae07-4591-a8de-57f0140c92b3.png)


***
пример корректно работающего mds. 
[HumanS.PsiCon.zip](https://github.com/dosinabox/g2nr_unofficial_update/files/9078843/HumanS.PsiCon.zip)

**Comment by Kor-Angar** [comment]
из минусов: при компиляции пришлось положить в папу `Hum_Body_Naked0.ASC `. скомпилированный файл Hum_Body_Naked0, использовать не стал и удалил.

**Comment by Kor-Angar** [comment]
в целом я рад, что оно работает так как наконец смогу вычленить недостающие оверлеи для Готики 1 и сложить их в отдельный архив.

**Comment by Kor-Angar** [comment]
получилось? :-)
[HUMANS.TELEKINESIS.zip](https://github.com/dosinabox/g2nr_unofficial_update/files/9087661/HUMANS.TELEKINESIS.zip)

**Reply by dosinabox** [reply]
получилось, спасибо

**Reply by Kor-Angar** [reply]
а я то что :-)

**Reply by Kor-Angar** [reply]
а 65 в телекинезе  обязательна ? или можно другую брать?

**Reply by dosinabox** [reply]
не помню, попробуй другую
